package.json

```diff
@@ -33,10 +33,11 @@
     "build:dockerimage_no_cache": "docker build -t mydockerimage --no-cache . ",
     "build:emscripten": "echo $(pwd) && docker run --rm -v $(pwd):/src -u $(id -u):$(id -g) mydockerimage copy_output.sh",
     "copy-files": "copyfiles -u 1 src/xpython_wasm.wasm src/xpython_wasm.worker.js src/xpython_wasm.js src/python_data.js src/python_data.data lib",
+    "copy-packagejson": "copyfiles package.json jupyterlite_xeus_python/",
```

Comment:
This is the only thing that I find annoying in this PR. Having to copy the `package.json` to be installed under `site-packages` feels dirty.

Reply:
Technically nothing stops us from having the version separated between the Python and the NPM package. That is what we do with jupyter-widgets. If we keep those versions separated, we don't need the `package.json` around the Python code anymore.

Reply:
Undone this `package.json`. It's not needed.